ZCRMiOS/Classes/ZCRMAnalytics.swift

```diff
@@ -0,0 +1,125 @@
+//
+//  ZCRMAnalytics.swift
+//  ZCRMiOS
+//
+//  Created by Kalyani shiva on 02/09/18.
+//
+
+import Foundation
+
+public class ZCRMAnalytics {
+    
+    public typealias dashBoard =
+        (Result.DataResponse<ZCRMDashBoard,APIResponse>) -> Void
+    
+    public typealias ArrayOfDashBoards = (Result.DataResponse<[ZCRMDashBoard],BulkAPIResponse>) -> Void
+    
+    public typealias dashBoardComponent = (Result.DataResponse<ZCRMDashBoardComponent,APIResponse>) -> Void
+    
+    public typealias refreshResponse = (Result.Response<APIResponse>) -> Void
+    
+    public typealias ArrayOfColorThemes = (Result.DataResponse<[ZCRMDashBoardComponentColorThemes],APIResponse>) -> Void
+    
+}
+
+//MARK:- Public Getters
+extension ZCRMAnalytics {
+    
+    public func getAllDashBoards(then onCompletion:@escaping ArrayOfDashBoards)
```

Comment:
use this as "Dashboard" instead of "DashBoard" - dashboard is a single word, so camel case doesn't come.

Reply:
Okay Anna